earth_enterprise/src/fusion/autoingest/tools/geaddtoimageryproject.cpp

```diff
@@ -88,6 +99,11 @@ main(int argc, char *argv[]) {
     options.opt("maxlevel", overridemax);
     options.opt("historical_imagery", enable_historical_imagery);
     options.opt("no_historical_imagery", disable_historical_imagery);
+    options.opt("terrain_overlay", enable_terrain_overlay);
+    options.opt("no_terrain_overlay", disable_terrain_overlay);
+    options.opt("start_level", start_level, &khGetopt::IsEvenNumberInRange<uint, 4, 24>); 
```

Comment:
instead of 4 and 24 magic numbers, let's add const with descriptive names.

Reply:
Will do.